draft-ietf-core-resource-directory.md

```diff
@@ -976,26 +980,25 @@ Plurality of link references exists if, and only if, two or more links in a Reso
 
 ## Operations on the Registration Resource
 
-After the initial registration, an endpoint should retain the returned location of the Registration Resource for further operations, including refreshing the registration in order to extend the lifetime and "keep-alive" the registration. When the lifetime of the registration has expired, the RD SHOULD NOT respond to discovery queries with information from the endpoint. The RD SHOULD continue to provide access to the Registration Resource after a registration time-out occurs in order to enable the registering endpoint to eventually refresh the registration. The RD MAY eventually remove the registration resource for the purpose of resource recovery and garbage collection. If the Registration Resource is removed, the endpoint will need to re-register.
+After the initial registration, an endpoint should retain the returned location of the Registration Resource for further operations, including refreshing the registration in order to extend the lifetime and "keep-alive" the registration. When the lifetime of the registration has expired, the RD SHOULD NOT respond to discovery queries concerning this endpoint. The RD SHOULD continue to provide access to the Registration Resource after a registration time-out occurs in order to enable the registering endpoint to eventually refresh the registration. The RD MAY eventually remove the registration resource for the purpose of resource recovery and garbage collection. If the Registration Resource is removed, the endpoint will need to re-register.
 
-The Registration Resource may also be used to inspect the registration resource using GET, update the registration link contents using iPATCH or PATCH (as introduced in {{RFC8132}}), or cancel the registration using DELETE.
+The Registration Resource may also be used to inspect the registration resource using GET, update the registration link contents, or cancel the registration using DELETE.
 
 These operations are described in this section.
 
 In accordance with {{link-plurality}}, operations which would result in plural link references within the context of a registration resource SHOULD be rejected using the "Conflict" result code.
 
-TODO: suggestion to do a GET /rd to obtain a list of registrations loc, ep,...
 
 ### Registration Update {#update}
 
 The update interface is used by an endpoint to refresh or update its
 registration with an RD. To use the interface, the endpoint sends a POST request to the registration resource returned in the Location header option in the response returned from the initial registration operation. The POST request (in contrast to PUT) allows replacing a selection of a resource.
 
-An update MAY update the lifetime or context registration parameters
+An update MAY update the lifetime- or the context- registration parameters
```

Comment:
We haven't typeset them as lifetime-registration parameters or similar; 'update the "lifetime" or the "context" registration parameters' perhaps?